### ISSUE TITLE
[Typescript] Could not find a declaration file for module

### ISSUE THREAD

**Issue by sajermann** [SAP/ui5-webcomponents#3460]
Hi,
I am converting my project for **Typescript**, and I having trouble in my imports:

```
import parse from '@ui5/webcomponents-base/dist/PropertiesFileFormat';
import { registerI18nLoader } from '@ui5/webcomponents-base/dist/asset-registries/i18n';
```
Error:
`Could not find a declaration file for module '@ui5/webcomponents-base/dist/PropertiesFileFormat'. 'c:/Users/b.sajermann.da.silva/Documents/Github/cockpittypescript/node_modules/@ui5/webcomponents-base/dist/PropertiesFileFormat.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/ui5__webcomponents-base` if it exists or add a new declaration (.d.ts) file containing `declare module '@ui5/webcomponents-base/dist/PropertiesFileFormat';`ts(7016)`;

Please, how can I resolve this?

**Comment by NHristov-sap** [comment]
Hi @SAP/ui5-webcomponents-team,

Can you please take a look at this?

Best Regards,
Nikolay Hristov

**Comment by pskelin** [comment]
Hi @sajermann ,
I was able to reproduce this and it is indeed quite annoying. Until the TS definitions we are working on are shipped (currently in progress, tracking #2470 ), I suggest the following workarounds:

**Option 1:**
create a file `index.d.ts` anywhere (or choose any meaningful name you like - `AnyTypes.d.ts`) and put the module declarations for the modules you are using:
```typescript
declare module '@ui5/webcomponents-base/dist/PropertiesFileFormat';
declare module '@ui5/webcomponents-base/dist/asset-registries/i18n';
```

This explicitly declares these modules as type `any` and the error will disappear. Once we ship types, all you need to do is delete this file.

**Option 2:**
use `ts-ignore` and add a descriptive reason what exactly the `ts-ignore` is for
```typescript
// @ts-ignore: no type definitions
import parse from '@ui5/webcomponents-base/dist/PropertiesFileFormat';
// @ts-ignore: no type definitions
import { registerI18nLoader } from '@ui5/webcomponents-base/dist/asset-registries/i18n';
```

Option 2 removes the error, but it has downside that it masks other errors (like variable never used). My personal preference is Option 1.

Please reopen the issue in case of further questions.
